src/util/checkError.ts

```diff
@@ -0,0 +1,24 @@
+import { t } from "i18next";
+
+/**
+ * Checks for the error message. Inconsistency in getting the actual error will be fixed with
+ * https://github.com/fusion44/blitz_api/issues/123
+ * @returns The error with the translated prefix "An error occurred" (error text not yet translated)
+ */
+export function checkError(err: any): string {
```

Comment:
Is it possible to not use `any`? Would `unknown` work?

Reply:
Will do typing, thank you ;)